OpenRA.Mods.RA/Scripting/Properties/InfiltrateProperties.cs

```diff
@@ -19,18 +21,21 @@ namespace OpenRA.Mods.RA.Scripting
 	[ScriptPropertyGroup("Ability")]
 	public class InfiltrateProperties : ScriptActorProperties, Requires<InfiltratesInfo>
 	{
-		readonly InfiltratesInfo info;
-
 		public InfiltrateProperties(ScriptContext context, Actor self)
-			: base(context, self)
-		{
-			info = Self.Info.TraitInfo<InfiltratesInfo>();
-		}
+			: base(context, self) { }
```

Comment:
> 00:05:44 (abcdefg30) wait, atlimit8` are you sure that we don't want to only cache TraitsImplementing and then evaluate the FirstOrDefault on the run?
00:05:56 (abcdefg30) because the TraitEnabled state could have changed in the meantime?